lizmap/modules/lizmap/classes/lizmapLogConfig.class.php

```diff
@@ -72,11 +108,11 @@ public function modify($data)
      *
      * @param array $data array containing the data of the general options
      */
-    public function update($data)
+    public function update($data, $ini)
```

Comment:
this change of this method annoy me, as it could break some modules if this method is used (it seems it is not used into Lizmap directly).

I propose to set this `$ini` parameter as optional : `update($data, $ini=null)`. Same for `save($ini=null)`. And in the `save()` method, if `$ini` is null, the jIniFileModifier should be instancied.